src/js/chrome/initialize-jwt.js

```diff
@@ -0,0 +1,34 @@
+import { headerLoader } from '../App/Header';
+import { allowUnauthed } from '../auth';
+import { decodeToken } from '../jwt/jwt';
+import { spinUpStore } from '../redux-config';
+import { CacheAdapter } from '../utils/cache';
+
+/**
+ * Initialize JWT
+ * @param {object} libjwt JWT API
+ * @param {object} chromeInstance persistent chrome object
+ * @returns {Promise}
+ */
+const initializeJWT = async (libjwt, chromeInstance) => {
+    let chromeCache;
+    const { actions } = spinUpStore();
+    try {
+        await libjwt.initPromise;
+        const user = await libjwt.jwt.getUserInfo();
+        actions.userLogIn(user);
+        chromeCache = new CacheAdapter(
+            'chrome-store',
+    `${decodeToken(libjwt.jwt.getEncodedToken())?.session_state}-chrome-store`
+        );
+        chromeInstance.cache = chromeCache;
+    } catch (_e) {
+        if (allowUnauthed()) {
+            actions.userLogIn(false);
+        }
+        chromeInstance.cache = chromeCache;
```

Comment:
This could lead to `chromeInstance.cache = undefined` perhaps moving cache init bellow `catch` would be better.

Reply:
Yeah.